yarn.lock

```diff
@@ -2403,10 +2403,10 @@
     webcrypto-shim "^0.1.5"
     xhr2 "0.1.3"
 
-"@okta/okta-idx-js@0.18.0":
-  version "0.18.0"
-  resolved "https://registry.yarnpkg.com/@okta/okta-idx-js/-/okta-idx-js-0.18.0.tgz#a419c915a68bcb925da33fe57a651826f681c57a"
-  integrity sha512-r6MZ7SMY/KwKVxDv0CuBR5QS5Iy6NsZCEC/zHqNpgXCd0GcKJPqrAt/xDWkp9oUccZrXI4PauU6zHnFSHLJEag==
+"@okta/okta-idx-js@0.19.0-g4c9be65":
+  version "0.19.0-g4c9be65"
+  resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-all/@okta/okta-idx-js/-/@okta/okta-idx-js-0.19.0-g4c9be65.tgz#ae9ec40d9630f9d9b820dc791833105b2f89099e"
```

Comment:
assume this will be updated to public registry when idx-js got promoted

Reply:
yes